core/src/main/resources/xml/xsl/esb/soapBody.xsl

```diff
@@ -141,22 +150,62 @@
 									</xsl:element>
 								</xsl:element>
 							</xsl:element>
-							<xsl:if test="string-length($errorDetailCode)&gt;0">
-								<xsl:element name="DetailList" namespace="{$ns}">
-									<xsl:element name="Detail" namespace="{$ns}">
-										<xsl:element name="Code" namespace="{$ns}">
-											<xsl:value-of select="$errorDetailCode"/>
-										</xsl:element>
-										<xsl:if test="string-length($errorDetailText)&gt;0">
+							<xsl:choose>
```

Comment:
De test op 'errorDetailCode' leek verdwenen te zijn, maar zag ik toch nog verder naar onderen. Wordt er ook getest dat de oude manier van werken ook nog werkt?

Reply:
Jazeker, dat zit in de testen van de adapter EsbSoapValidator